src/translator/beam_search.h

```diff
@@ -155,7 +155,11 @@ class BeamSearch {
       return;
     for(auto node : *currTrieNode) {
       auto index = node.id_ + rowNum*nColumns; //node.id_ is a vocab ID
+      //if (in[index] >= -15){
+        //std::cout << in[index] << " | ";
         in[index] += bumpVal;
+        //std::cout << in[index] << " || ";
```

Comment:
Can you remove the commented out printing code here and above

Reply:
done as per review.